.gitignore

```diff
@@ -17,3 +17,4 @@ dist/
 
 # ide
 .vscode/
+.idea/
```

Comment:
I would rather remove developer-specific lines like there which are probably more fitting for [`.git/info/exclude`](https://stackoverflow.com/a/1753078/939364).

Reply:
I'm agree with you (it's hard to maintain that for all IDEs) and I have deleted both entries. However, it can be found in other Scrapy projects:
* https://github.com/scrapy/cssselect/blob/master/.gitignore#L8
* https://github.com/scrapy/scrapy/blob/master/.gitignore#L12

So, maybe it could be a good idea to standarize all gitignores or open an issue/PR.